commerce_coordinator/apps/commercetools/utils.py

```diff
@@ -144,3 +147,87 @@ def translate_stripe_refund_status_to_transaction_status(stripe_refund_status: s
         'failed': TransactionState.FAILURE,
     }
     return translations.get(stripe_refund_status.lower(), stripe_refund_status)
+
+
+def send_refund_notification(user, order_id):
+    """
+    Notify the support team of the refund request.
+
+    Returns:
+        bool: True if we are able to send the notification.  In this case that means we were able to create
+              a ZenDesk ticket
+    """
+
+    tags = ['auto_refund']
+
+    # Build the information for the ZenDesk ticket
+    student = user
+    subject = "[Refund] User-Requested Refund"
+    body = generate_refund_notification_body(student, order_id)
+    requester_name = student.username
+
+    return create_zendesk_ticket(requester_name, student.email, subject, body, tags)
+
+
+def generate_refund_notification_body(student, order_id):
+    """ Returns a refund notification message body."""
+
+    msg = f"""A refund request has been initiated for {student.username} ({student.email}).
```

Comment:
Please see my comment above on retrieving the username from the CT customer class.